test/Microsoft.ML.Benchmarks/FeaturizeTextBench.cs

```diff
@@ -0,0 +1,136 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
@sharwell Please, review this benchmark test and tell me if this was what you hand in mind when you requested to add one